### PR TITLE
ENH: Added bad non-binary letter

### PR DESCRIPTION
**Pull request by mjonesjr**
Added bad non-binary letter to the references on the webpage and added it to example letters. Partially fulfills Issue #40

### CODE REVIEW

File: elspSampleLetters.html [hunk 1]
```diff
@@ -233,6 +233,30 @@ <h2>Sample Recommendation Letters</h2>
                     </div>
                 </div>
 
+                <button class="expandCollapse"> <i class="em em-anguished" aria-role="presentation" aria-label="Sad"></i> Recommendation letter: unsupporitve letter for a post graduate non-binary candidate.</button>
+                <div id="letter7" class="sampleLetter">
+                    <div class="letterContent">
+                        <img class="letterhead" src="images/impressiveLetterhead.jpg">
+
+                        <p><span>class="hoverText" title="Throughout the letter, the writer uses they/their pronouns incorrectly. Use they/their just as you would use she/her or he/him. We’ve bolded the points below that the pronouns should be used better.">Dear Professor Lee:</p>
```
Comment (aburrell) [inline elspSampleLetters.html:241]:
```suggestion
                        <p><span class="hoverText" title="Throughout the letter, the writer uses they/their pronouns incorrectly. Use they/their just as you would use she/her or he/him. We’ve bolded the points below that the pronouns should be used better.">Dear Professor Lee:</p>
```

**Review by aburrell — COMMENTED** [review]
Found a bug.

**Review by aburrell — APPROVED** [review]
I fixed the issues through a commit.  I think it's ready to merge, but you should take a brief look to confirm.

**Comment by mjonesjr** [comment]
Looks good! Thanks for catching these typos!

**Pull request merged**